packages/applications/cli/src/commands/abandon/relancer-abandon-sans-preuve.ts

```diff
@@ -0,0 +1,90 @@
+import { mediator } from 'mediateur';
+import { Command } from '@oclif/core';
+import z from 'zod';
+
+import { DateTime } from '@potentiel-domain/common';
+import {
+  Abandon,
+  registerLauréatQueries,
+  registerLauréatUseCases,
+} from '@potentiel-domain/laureat';
+import {
+  AppelOffreAdapter,
+  DocumentAdapter,
+  récupérerIdentifiantsProjetParEmailPorteurAdapter,
+} from '@potentiel-infrastructure/domain-adapters';
+import { loadAggregate, loadAggregateV2 } from '@potentiel-infrastructure/pg-event-sourcing';
+import {
+  countProjection,
+  findProjection,
+  listProjection,
+} from '@potentiel-infrastructure/pg-projection-read';
+import { getLogger } from '@potentiel-libraries/monitoring';
+import { ProjetAggregateRoot } from '@potentiel-domain/projet';
+import { killPool } from '@potentiel-libraries/pg-helpers';
+
+const envSchema = z.object({
+  APPLICATION_STAGE: z.string(),
+  DATABASE_CONNECTION_STRING: z.string().url(),
+});
+export class Relancer extends Command {
+  async init() {
+    const { APPLICATION_STAGE } = envSchema.parse(process.env);
+    if (!['production', 'development'].includes(APPLICATION_STAGE)) {
```

Comment:
```suggestion
    if (!['production', 'local'].includes(APPLICATION_STAGE)) {
```

On utilise `local` pour notre env de developpement